target/scripts/start-mailserver.sh

```diff
@@ -1970,42 +1969,45 @@ function misc
 
 function _misc_save_states
 {
-  # consolidate all states into a single directory (`/var/mail-state`) to allow persistence using docker volumes
-  statedir=/var/mail-state
+  # consolidate all states into a single directory (`/var/mail-state`)
+  # to allow persistence using docker volumes
 
-  if [[ ${ONE_DIR} -eq 1 ]] && [[ -d ${statedir} ]]
+  local STATEDIR=/var/mail-state
```

Comment:
Quotes missing. Not necesary, but to keep one standard across the whole file.

STATEDIR="/var/mail-state"

Reply:
True. I think this is okay for now, and I will push this with a future commit.